processes/ingest/hathi_trust.py

```diff
@@ -36,44 +37,50 @@ def runProcess(self):
 
         logger.info(f'Ingested {len(self.records)} Hathi Trust records')
 
-    def importRemoteRecords(self, fullOrPartial=False):
-        self.importFromHathiTrustDataFile(fullDump=fullOrPartial)
+    def importRemoteRecords(self, start_date_time=None, full_or_partial=False):
+        self.importFromHathiTrustDataFile(start_date_time, full_dump=full_or_partial)
 
-    def importFromSpecificFile(self, filePath):
+    def importFromSpecificFile(self, file_path):
         try:
-            hathiFile = open(filePath, newline='')
+            hathi_file = open(file_path, new_line='')
         except FileNotFoundError:
             raise IOError('Unable to open local CSV file')
 
-        hathiReader = csv.reader(hathiFile, delimiter='\t')
+        hathiReader = csv.reader(hathi_file, delimiter='\t')
         self.readHathiFile(hathiReader)
 
-    def parseHathiDataRow(self, dataRow):
-        hathiRec = HathiMapping(dataRow, self.statics)
+    def parseHathiDataRow(self, data_row):
+        hathiRec = HathiMapping(data_row, self.statics)
         hathiRec.applyMapping()
         self.addDCDWToUpdateList(hathiRec)
 
 
-    def importFromHathiTrustDataFile(self, fullDump=False):
+    def importFromHathiTrustDataFile(self, start_date_time=None, full_dump=False):
         try:
-            fileList = requests.get(os.environ['HATHI_DATAFILES'], timeout=15)
-            fileList.raise_for_status()
+            file_list = requests.get(os.environ['HATHI_DATAFILES'], timeout=15)
+            file_list.raise_for_status()
         except (ReadTimeout, HTTPError):
             raise IOError('Unable to load data files')
 
-        fileJSON = fileList.json()
+        file_json = file_list.json()
 
-        fileJSON.sort(
+        file_json.sort(
             key=lambda x: datetime.strptime(
                 x['created'],
                 self.returnHathiDateFormat(x['created'])
             ).timestamp(),
             reverse=True
         )
 
-        for hathiFile in fileJSON:
-            if hathiFile['full'] == fullDump:
-                self.importFromHathiFile(hathiFile['url'])
+        for hathi_file in file_json:
+            if hathi_file.get('full') == full_dump and hathi_file.get('full') == False:
```

Comment:
can you simplify this to`not hathi_file.get('full') && not full_dump`?

Reply:
Oh I get what you mean. That's a cleaner way to explain this same logic.